executor/common_linux.h

```diff
@@ -251,8 +253,13 @@ static void initialize_netdevices(void)
 	if (!flag_enable_net_dev)
 		return;
 #endif
+#if SYZ_EXECUTOR || SYZ_ENABLE_NETDEV_DEVTYPES || SYZ_ENABLE_NETDEV_DEVNAMES || SYZ_ENABLE_NETDEV_DEVMASTERS
 	unsigned i;
+#endif
+#if SYZ_EXECUTOR || SYZ_ENABLE_NETDEV_DEVTYPES
```

Comment:
If DEVTYPES is not enabled, then we won't have "ip6gretap0", "bridge0", "vcan0", "bond0", "team0", "veth0", "veth1" devices. I think we should remove them and don't try to setup.